packages/firebase_remote_config/lib/src/remote_config.dart

```diff
@@ -0,0 +1,210 @@
+part of firebase_remote_config;
+
+/// LastFetchStatus defines the possible status values of the last fetch.
+enum LastFetchStatus { success, failure, throttled, noFetchYet }
+
+class FetchThrottledException implements Exception {
```

Comment:
My vote is to split each class into a separate file in src/

Reply:
Done